docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,531 @@
+
+    ```
+    kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+    ```
+
+All done!
+
+#### Starting fresh install on a cluster 
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI version*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ dapr status -k
+
+  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
+  dapr-dashboard         dapr-system  True     Running  1         0.3.0    15s  2020-09-28 13:07.39
+  dapr-sidecar-injector  dapr-system  True     Running  1         0.11.0   15s  2020-09-28 13:07.39
+  dapr-sentry            dapr-system  True     Running  1         0.11.0   15s  2020-09-28 13:07.39
+  dapr-operator          dapr-system  True     Running  1         0.11.0   15s  2020-09-28 13:07.39
+  dapr-placement         dapr-system  True     Running  1         0.11.0   15s  2020-09-28 13:07.39
+```
+
+After Dapr 0.11.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### Serialization changes for gRPC State Transaction APIs
+
+A double serialization issue when gRPC State Transaction API was used, has been fixed in this release. As a result, states previously saved via the gRPC State Transaction API, will not be properly deserialized with the latest v0.11.0 version of the code. 
+
+For example previously, a string saved using the gRPC State Transaction API was saved as: 
+
+```
+\"saved state\"
+```
+
+When deserialized, it would be read as `"saved state"` instead of `saved state`. Similarly for JSON serialized values, it was marshaled once again and then saved.
+
+This has been fixed in v0.11.0 and the double serialization removed. Any code that expects the value in the previous incorrectly serialized format will need some changes to work with v0.11.0.
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component
```

Comment:
Can we remove this entire section as it's not a user facing change?

Reply:
We we remove this implementation details and point to the detailed issue "Read more [here](https://github.com/dapr/dapr/issues/2016). " so that people can find out more as a component developer